docker-traceability-plugin/src/main/resources/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityRootAction/_api.jelly

```diff
@@ -25,200 +25,227 @@
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
+<div id="docker-traceability">
+  <style scoped="true">
+    /* scoped only works in FF therefore we have to add a wrapper (div#docker-traceability) */
```

Comment:
Weak :-1: on this CSS in the plugin.  If you want to add specific highlighting, it's better to create appropriate controls in Jenkins core. These styles may be an interim solution only